### PR TITLE
e2e test fixes

### LINKED ISSUE #21593: e2e: fix most commonly failing test suites
Context: https://github.com/sourcegraph/sourcegraph/issues/21249#issuecomment-848818178

Most commonly failing tests (taken from above link):
- Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/sourcegraph/client/web/src/end-to-end/end-to-end.test.ts)
  - Test: `External service add, edit, delete`
- Error: Node is either not visible or not an HTMLElement
  - Tests: `Blob - jump to definition - does navigation (same repo, different file)`, `Repository component - revision resolution - updates revision with switcher`

### PR DESCRIPTION
**Pull request by novoselrok**
Fixes #21593

### CODE REVIEW

**Review by valerybugakov — APPROVED** [review]
Thanks for looking into this!

File: client/web/src/end-to-end/end-to-end.test.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ describe('e2e test suite', () => {
         ]
         const alwaysCloningRepoSlugs = ['sourcegraphtest/AlwaysCloningTest']
         await driver.ensureLoggedIn({ username: 'test', password: config.testUserPassword, email: 'test@test.com' })
+        await driver.page.evaluate(() => localStorage.setItem('isRedesignEnabled', 'true'))
```
[the remark this answers is not in the document]
Comment (novoselrok) [reply client/web/src/end-to-end/end-to-end.test.ts:54]: Redesign is already turned on by default if no previous value is set. With redesign enabled, it broke a couple of E2E tests and I didn't want to fix both cases. Here I wanted to make it clear that the E2E tests only work with the redesign enabled.

I'll refactor to use the `toggleRedesign` function.

[the remark this answers is not in the document]
Comment (valerybugakov) [reply client/web/src/end-to-end/end-to-end.test.ts]: Thanks!

**Review by umpox — APPROVED** [review]
🙌

File: client/web/src/end-to-end/end-to-end.test.ts [hunk 2]
```diff
@@ -113,7 +122,8 @@ describe('e2e test suite', () => {
             }, message)
         })
 
-        test('Check access tokens work (create, use and delete)', async () => {
+        // TODO: Fails locally with `RequestError: connect ECONNREFUSED 127.0.0.1:443`
```
Comment (felixfbecker) [inline client/web/src/end-to-end/end-to-end.test.ts:125]: Probably an issue with Caddy's magic HTTPS + Node not being able to reach it. Maybe Caddy only registers the certificates with browsers, but not with Node. But it's also suspicious that it's an IP and not a host name, for HTTPS?

**Review by eseliger — APPROVED** [review]
🙏

File: client/web/src/end-to-end/end-to-end.test.ts (hunk 2 above)
Comment (unknwon) [reply client/web/src/end-to-end/end-to-end.test.ts:125]: Not sure if this is your question, HTTP is default to port 80, HTTPS is default to port 443, browser is making URL looks clean for users.

**Review by unknwon — APPROVED** [review]

**Pull request merged**